packages/@guardian/source-foundations/src/utils/resets.test.ts

```diff
@@ -0,0 +1,56 @@
+import { transform } from 'lightningcss';
+import { resets } from './resets';
+
+it('should provide valid CSS for resets.defaults', () => {
+	expect(() =>
+		transform({ code: Buffer.from(resets.defaults, 'utf8'), filename: '' }),
+	).not.toThrowError();
+});
+
+it('should provide valid CSS for resets.fieldset', () => {
+	expect(() =>
+		transform({
+			code: Buffer.from(
+				`* {
+					${resets.fieldset}
+				}`,
+				'utf8',
+			),
+			filename: '',
+		}),
+	).not.toThrowError();
+});
+
+it('should provide SCSS for resets.input', () => {
```

Comment:
we expect this to error, because for this case we export SCSS — but fixing this will be more involved and arguably beyond the scope of this work

Reply:
I added a comment above that css block where it’s defined to make this intention clear

Reply:
is there a similar package we can use to verify that it's valid scss?

Reply:
given that scss is a superset of css, being invalid css is not proof that we have valid scss :) it could be invalid css _and_ scss

Reply:
hahaha, yes good point. 
Although useful for cases like this.. My immediate thought is that by adding `xyzSCSS` are we implicitly saying that SCSS is okay to go ahead and use in `source-foundations`?
If this is something we don't want, we might want to signal that quite strongly

Reply:
i guess the alternative is to mask that something _is_ scss, and therefore may break if you use it as css, which seems like a bad situation to me!

i'm not sure we _should_ be providing scss, but given that currently we _do_, making sure it's only used in the correct places has value.

we could mark it as `@deprecated` if you like, so at least we can convey that the use it not encouraged

Reply:
we have a pattern for this in `react-components` https://github.com/guardian/source/blob/main/packages/%40guardian/source-react-components/src/deprecated-exports.ts

Reply:
I've added a `deprecated` rule above this